test/Common.FunctionalTests/Inprocess/FrebTests.cs

```diff
@@ -96,23 +110,32 @@ private async Task<IISDeploymentResult> SetupFrebApp()
             return result;
         }
 
-        private void AssertFrebLogs(IISDeploymentResult result, params string[] data)
+        private void AssertSingleFrebLog(IISDeploymentResult result, params string[] data)
```

Comment:
Load all logs into an object structure and assert against it.

Reply:
Add a class FrebLog with FrebLogItem[] inside. Then you can use Assert.Contains and friends.

Reply:
Create a FrebLogItem class, but didn't add a wrapper class.